fix(curriculum):Added return inside map from step-25 to 45 and test for checking space between isCaptain and name (#56918)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-modern-javascript-methods-by-building-football-team-cards/63e954321b0a77ae4f6d9650.md

# --description--

> Inside the body of the callback function, you will need to return template literals ``` `` ``` which will contain the HTML content for the player cards. 

Reviewer: ```suggestion
Inside the body of the callback function, you will need to return a template literal ``` `` ``` which will contain the HTML content for the player cards. 
```

Reviewer 2: done

Inside the template literals, add an empty `div` with a class of `"player-card"`.

# --hints--

> You should use template literals to add an empty `div` with a class of `"player-card"` and `return` them.

Reviewer: ```suggestion
The `map` callback should return a template literal that containes an empty `div` with a class of `"player-card"`.
```

Reviewer 2: done
